src/common/datomish/query/clauses.cljc

```diff
@@ -6,6 +6,7 @@
   (:require
    [datomish.query.cc :as cc]
    [datomish.query.functions :as functions]
+   [datomish.query.projection :refer [sql-projection-for-simple-variable-list]]
```

Comment:
Consider folding the previous commit into this one; it's factored but hard to understand without a use (or test).